File: Java/Medium/Accepted/maximum-number-of-consecutive-values-you-can-make_3.java
class Solution {
    public int getMaximumConsecutive(int[] coins) {
        
        Arrays.sort(coins);
        int max = 0;
        for (int i = 0; i < coins.length; i++) {
            if (coins[i] <= max + 1) {
                max += coins[i];
            } else {
                break;
            }
        }
        return max + 1;
    }
}

File: Rust/Medium/Accepted/maximum-number-of-consecutive-values-you-can-make_3.rs
impl Solution {
    pub fn get_maximum_consecutive(coins: Vec<i32>) -> i32 {
        
        let mut coins = coins;
        coins.sort();
        let mut res = 0;
        for c in coins {
            if c > res + 1 {
                break;
            }
            res += c;
        }
        res + 1
    }
}